.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```

Comment:
Should we add a new feature_request label?

Reply:
I used `enhancement` here, which already exists.